lib/stages/visAccumulate.hpp

```diff
@@ -157,6 +162,7 @@ class visAccumulate : public kotekan::Stage {
     size_t num_gpu_frames;
     size_t minimum_samples;
     float max_age;
+    dset_id_t fpga_dataset;
```

Comment:
Config variables should start with an underscore (http://lwlab.dunlap.utoronto.ca/kotekan/dev/dev_style_guide_code.html#variables).

Reply:
Indeed. But none of the existing ones do, so I'm going for consistency rather than adherence to the guidelines in here.